distribution/src/main/resources/drill-override-example.conf

```diff
@@ -176,6 +176,15 @@ drill.exec: {
           validatePeerCerts: false,
           # true if SSL wants client authentication.
           wantClientAuth: false
+        },
+        response: {
+          # any response headers with constant values may be configured like this
+          headers: {
+            "X-XSS-Protection": "1; mode=block",
+            "X-Content-Type-Options": "nosniff",
+            "Strict-Transport-Security": "max-age=31536000;includeSubDomains",
+            "Content-Security-Policy": "default-src https:; script-src 'unsafe-inline' https:; style-src 'unsafe-inline' https:; font-src data: https:; img-src data: https:"
```

Comment:
Could you please create Jira to remove inline from scripts and styles and leave comment here with reference to this Jira indicating why we have to allow inline for now. Thanks.

Reply:
Thanks for noticing it. Created new ticket [DRILL-7642](https://issues.apache.org/jira/browse/DRILL-7642).

Reply:
Thanks. Please don't forget to add comment as well :)

Reply:
done

Reply:
Perfect, thanks.